modules/helper.js

```diff
@@ -36,8 +36,12 @@ class Helper {
         return os.release().includes(WIN_WSL_OS_RELEASE);
     }
 
-    static confirmNodeProvider(tezosNode) {
-        return tezosNode.includes('localhost');
+    static isCarthagenetNode(tezosNode) {
```

Comment:
change method name to something more generic or meaningful. 
For every new protocol upgrade, you can't change method name.
don't hardcode protocol name anywhere in the code base. 
Find a better way to achieve this.